### DISCUSSION TITLE
Colour shift problem when using regional prompt feature

### DISCUSSION THREAD

**Discussion by mysticfall** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#255] [Q&A]
I find the regional prompt this extension provides to be essential in my toolbox since it's one of the very few ways I can compose a complex scene without destroying the overall consistency.

However, I found that it tends to heavily affect the tone and lighting of the scene, sometimes making it look strongly yellow-tinted regardless of whatever I do to counter it.

Is there any way I can mitigate the problem?

**Comment by Rkkss** [comment]
Use different vae
